controllers/infra/controllers/infra_controller.go

```diff
@@ -68,13 +67,35 @@ func (r *InfraReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 		return ctrl.Result{}, err
 	}
 
+	//status
+	var cstZone = time.FixedZone("CST", 8*3600)
+	infra.Status.CreateTime = time.Now().In(cstZone).Format("2006-01-02 15:04:05")
+	infra.Status.RegionIDs = infra.Spec.RegionIDs
```

Comment:
If the spec already has RegionID, Why does status need to add this field repeatedly